docs/src/pages/guides/notifications.md

```diff
@@ -1,3 +1,214 @@
-# Notifications
+---
+id: notifications
+title: Notifications
+---
 
-under construction
+## Notification Types
+
+In `Monika`, we provide multiple notification channels. Notifications are available through sendgrid, mailgun, smtp and webhook.
+
+## Configurations
+
+Notifications can be sent through multiple channels and mail accounts, each are defined through configuration such as in the file `config.json.example` below. All you need for having a notification is set these configurations.
+
+```json
+"notifications": [
+    {
+      "id": "unique-id",
+      "type": "mailgun",
+      "data": {
+        "recipients": ["mantap@jiwa.com"],
+        "apiKey": "YOUR_API_KEY",
+        "domain": "YOUR_DOMAIN"
+      }
+    },
+    {
+      "id": "unique-id",
+      "type": "sendgrid",
+      "data": {
+        "recipients": ["mantap@jiwa.com"],
+        "apiKey": "YOUR_API_KEY"
+      }
+    },
+    {
+      "id": "unique-id",
+      "type": "smtp",
+      "recipients": ["a@a.com"],
+      "data": {
+        "recipients": ["mantap@jiwa.com"],
+        "hostname": "https://www.dennypradipta.com",
+        "port": 8080,
+        "username": "dennypradipta",
+        "password": "bismillah"
+      }
+    },
+    {
+      "id": "unique-id-webhook",
+      "type": "webhook",
+      "data": {
+        "recipients": ["mantap@jiwa.com"],
+        "method": "POST",
+        "url": "https://examplewebhookurl.com/webhook"
+      }
+    }
+  ],
+```
+
+## Mailgun
+
+Mailgun is the notification channel through the mailgun email service. For this you need to have a mailgun account first, using your mailgun's API_KEY and DOMAIN you can send notifications to the recipient's mail.
+
+```json
+{
+  "id": "unique-id",
+  "type": "mailgun",
+  "data": {
+    "recipients": ["mantap@jiwa.com"],
+    "apiKey": "YOUR_API_KEY",
+    "domain": "YOUR_DOMAIN"
+  }
+}
+```
+
+**ID**
+Notification ID, to identify the notification number.
+
+**Type**
+Type of Notification in this case it would be set as `mailgun`.
+
+**Data**
+The detail settings needed to send a mailgun.
+
+**Recipients**
+Email addresses that will accept the mail sent from your mailgun account, the recipients address is set in array making it able to send to multiple addresses. `["monika@testmail.com", "symon@testmail.com"]`
+
+**Api Key**
+Mailgun's account api key, mailgun's registered key to identify your account. ex. `70e34aba-0ea908325`
+
+**Domain**
+A domain that you have set in mailgun. ex. `sandboxmail.mailgun.com`
+
+`Monika` filters the notification types for mailgun and breaks down the data configuration, setting up a mailgun service and then sending through the data using mailgun-js lib.
+
+## Sendgrid
+
+Similar to mailgun, sendgrid is also an email delivery service. You need to have a sendgrid account before you can send an email through this method. For sending a notification through sendgrid all you need to set is the API_KEY and recipient email addresses.
```

Comment:
Similar to mailgun, sendgrid is also an email delivery service. First, sign up for an account in sendgrid and get the API key. Please check [sendgrid's documentation](https://sendgrid.com/docs/api-reference/). Then put the API key in Monika's configuration as follows.